style.md

```diff
@@ -339,7 +339,7 @@ A single blank line appears:
 
  2. Between statements, as _needed_ to organize the code into logical subsections.
 
- 3. _Optionally_ before the first statement in a function, or before the first member of after the last member of a class (neither encourage nor discouraged).
+ 3. _Optionally_ before the first statement in a function, or either before the first member or after the last member of a class (neither encourage nor discouraged).
```

Comment:
Sorry to nitpick on this again, but "either" makes it sounds like you _can't_ have both.

Maybe:

> _Optionally_ before the first statement in a function, before the first member of a class, or after the last member of a class (neither encouraged or discouraged).

Reply:
Ok, I didn't know that "either" is XOR (I thought that it is OR) :) You're right... so that's the only right way. Just applied your suggestion in the next commit.